logo/dockbit.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="38px" height="38px" viewBox="0 0 38 38" enable-background="new 0 0 38 38" xml:space="preserve">
+<path fill="#FFFFFF" d="M29.073,9.964L18.898,4.091c-1.008-0.583-2.25-0.583-3.259,0L5.465,9.964
+	c-1.008,0.582-1.629,1.658-1.629,2.822v11.749c0,1.164,0.621,2.24,1.629,2.82l10.175,5.877c1.008,0.582,2.251,0.582,3.259,0
+	l10.175-5.877c1.008-0.58,1.628-1.656,1.628-2.82V12.786C30.701,11.623,30.081,10.546,29.073,9.964 M28.095,24.229
+	c0,0.35-0.187,0.672-0.487,0.846l-2.381,1.377c-0.303,0.174-0.675,0.174-0.978,0l-2.383-1.377c-0.301-0.172-0.488-0.496-0.488-0.846
+	V21.48c0-0.35,0.188-0.672,0.488-0.848l2.383-1.374c0.303-0.176,0.675-0.176,0.978,0l2.381,1.374
+	c0.301,0.176,0.487,0.498,0.487,0.848V24.229z"/>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```

Comment:
Please remove the empty `<g>` elements. They don't do anything when they're empty  :)